ui/src/js/viewModels/orgAdmin.js

```diff
@@ -7,436 +7,453 @@
-                                "user_id":self.userId(),
-                                "organization_id":self.userOrgId(),
-                                "admin": (self.userAdmin().length > 0) ? "Y" : "N",
-                                "need_approver": (self.userNeedApprover().length > 0) ? "Y" : "N",
-                                "user_approver": (self.userApprover().length > 0) ? "Y" : "N",
-                                "manage_clients": (self.userManageClients().length > 0) ? "Y" : "N",
-                                "manage_offers": (self.userManageOffers().length > 0) ? "Y" : "N",
-                                "client_share_approver": (self.userClientShareApprover().length > 0) ? "Y" : "N",
-                                "confirmed": (self.userConfirmed().length > 0) ? "Y" : "N"
-                            };
-                        return $.when(restClient.doPost('/rest/users', userData)
-                            .then(
-                                success = function (response) {
-                                    self.postText("You have succesfully saved user details.");
-                                    self.postTextColor("green");
-                                    return $.when(restClient.doPost('/rest/organizations/' + self.userOrgId() + '/user_organizations', userOrgData)
-                                        .then(
-                                            success = function (response) {
-                                                self.getUserOrgData(self.userOrgId());
-                                                console.log("user data posted");
-                                            })
-                                    );
-                                },
-                                error = function (response) {
-                                    self.postText("Error: User changes not saved.");
-                                    self.postTextColor("red");
-                                    console.log("user data not posted");
-                                }).then(function () {
+                    /*
+                    var userOrgData =
+                    {
+                        "id": self.id(),
+                        "user_id": self.userId(),
+                        "organization_id": self.userOrgId(),
+                        "admin": (self.userAdmin().length > 0) ? "Y" : "N",
+                        "need_approver": (self.userNeedApprover().length > 0) ? "Y" : "N",
+                        "user_approver": (self.userApprover().length > 0) ? "Y" : "N",
+                        "manage_clients": (self.userManageClients().length > 0) ? "Y" : "N",
+                        "manage_offers": (self.userManageOffers().length > 0) ? "Y" : "N",
+                        "client_share_approver": (self.userClientShareApprover().length > 0) ? "Y" : "N",
+                        "dbs_check": (self.userDbsCheck()) ? "U" : "U",
```

Comment:
This looks odd- we are checking if self.userDbsCheck() is true, and if it is, we set dbs_check to U. If it is not, we also set it to U.

Reply:
I have changed it to the check we use in user.js although as this is part of the commented out section it doesn't affect the code.